src/connectors/numeric-menu/connectNumericMenu.ts

```diff
@@ -323,6 +328,7 @@ const connectNumericMenu: NumericMenuConnector = function connectNumericMenu(
           createURL: connectorState.createURL!(state),
           items: transformItems(prepareItems(state)),
           hasNoResults: results ? results.nbHits === 0 : true,
+          canRefine: Boolean(results && results.nbHits > 0),
```

Comment:
maybe `items.every(({count}) => count > 0)`?

Reply:
every vs some?

Reply:
both every and some shortcut as soon as the condition is fulfilled or not fulfilled so it's not that big of a difference, the condition just needs to be inverted

Reply:
I'm looking at it again, and there is no count available: https://github.com/algolia/instantsearch.js/pull/4688/files#diff-9dc3db9af2e7feeb52390473e9a560153d8b66bf8a7b358d1041f52b74ff610cR35-R53

since it's an array made by user, we cannot know how many hits each item contains ahead.

so, what about just `items.length > 0`?

Reply:
Actually that was intended to check the items after `transformItems`, but we shouldn't need to cover whatever happens in the userland. If we do it here, then we should do it in sortBy, too.
So I think I should just close this PR. What do you think?